docs/mine/spacerace.md

```diff
@@ -187,6 +187,26 @@ lotus-miner sectors status --on-chain-info $SECTOR_NUMBER | grep OnTime
 
 - If the deal is marked as successful on the dashboard but the above commands do not return the expected result, simply repeat steps #1-3 above.
 
+#### How do I force a WaitDeals sector to start sealing?
+
+Run `lotus-miner sectors seal <sector id>`.
+
+#### How is the deal rate of the combined miners being calculated?
+
+The deal rate is calculated based on the sum of all the deals across the miners.
+
+#### If I rent a server in EU from US, can I compete in EU region?
+
+The definition of the `mining operation` can be found [here](https://spacerace.filecoin.io/rules), in short, it is the physical location of the storage and sealing hardware. It might not be cost effective, but you can rent cloud service anywhere in the world. However, it is NOT acceptable to do something like running a proxy or anything of that nature.
+
+#### Why is my deal success rate below 80%?
+
+Deals can fail due to various reason. Go to [Space Race Dashboard](https://spacerace.filecoin.io/), search for your miner id and click `Open Logs` to see the errors that caused the failures.
+
+#### My sector is stuck in X stage, how can I remove it?
+
+After Lotus version v0.5.6, you can remove a sector by running `lotus-miner sectors remove --really-do-it <sector id>`. Warning: by doing this means losing power and collateral for the removed sector.
```

Comment:
```suggestion
In lotus v0.5.6 and later, you can remove a sector by running `lotus-miner sectors remove --really-do-it <sector id>`. Warning: This will cause you to lose all power and collateral for the removed sector.
```